ibrahim/styles.css

```
@@ -0,0 +1,270 @@
+* {
+  box-sizing: border-box;
+}
+
+/* Undoing defaults */
+body {
+  margin: 0 0 0 0;
+  transition: all 750ms;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+figure {
+  margin: 0;
+}
+
+img {
+  height: auto;
+  width: 100%;
+}
+
+/* ---------- */
+
+body {
+  font-family: "Quicksand", sans-serif;
+  color: #1e1e1e;
+  font-size: 10px;
+  background: linear-gradient(#f0f0f0, #ffffff);
+}
+
+.text {
+  margin: 20px 25px;
+}
+
+h1 {
+  color: #015759;
+  font-family: "Comfortaa", sans-serif;
+  font-size: 36px;
+}
+
+h2 {
+  color: #015759;
+  font-family: "Comfortaa", sans-serif;
+  font-size: 18px;
+}
+
+/* Navigation */
+nav {
+  background-color: #1e1e1e;
+  text-align: center;
+}
+
+nav li {
+  display: inline-block;
+  margin-left: -3px;
+}
+
+nav .first-item {
```

You actually don't need a class to do this; it's built into CSS!
If you change your selector to this, it will accomplish what you want and you can delete the classname:
```suggestion
nav ul li:first-child {
```

`:first-child` is another psuedoclass, like `:hover` and `:focus`; we didn't cover it in class, but I'm pointing it out here since it's relevant to what you want to accomplish.